client/app/visualizations/choropleth/Renderer/index.jsx

```diff
@@ -58,7 +58,7 @@ export default function Renderer({ data, options, onOptionsChange }) {
         options, // detect changes for all options except bounds, but pass them all!
       );
     }
-  }, [map, geoJson, data, optionsWithoutBounds]);
+  }, [map, geoJson, data, optionsWithoutBounds, options]);
```

Comment:
This change will cause performance issues - see comment above

Reply:
@gabrieldutra Can you please address this in #4425 as well?

Reply:
Done :+1: